community_topics_workflow.md

```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
 
 The Workflow is a set of actions that need to be done successively within the corresponding time frames.
 
-After a topic is created, a Steering Committee member (hereinafter `Committee person`) copies the Workflow into the topic and, along the way, marks the checkboxes as complete to track the progress, see and plan the following actions needed to get the topic resolved within acceptable time.
-
 **Note:** If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
 
+## Creating a topic
+
+Any user can [created a topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=steering-committee) tagged with `steering-committee` under the `Project Discussions` category in the [Ansible Forum](https://forum.ansible.com/). A steering committee member can tag the forum post with `community-meeting` to put it on the meeting agenda.
```

Comment:
```suggestion
Any user can [create a topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=steering-committee) tagged with `steering-committee` under the `Project Discussions` category in the [Ansible Forum](https://forum.ansible.com/). A steering committee member can tag the forum post with `community-meeting` to put it on the meeting agenda.
```

Reply:
@gotmax23 thanks for spotting, merged, PTAL